redmine

```diff
@@ -40,7 +40,7 @@ def set_status_version(args: argparse.Namespace) -> None:
     redmine = get_redmine()
     versions = redmine.version.filter(project_id=args.project_id)
     try:
-        version = [version for version in versions if version.name == args.name][0]
+        version = [version for version in versions if args.name in version.name][0]
```

Comment:
I'd like to avoid `in` checks with version numbers. You can easily match `3.8.10` when you intended `3.8.1`. How about splitting on space and assuming the last arg is the version?

Is this too hard to read?
```suggestion
        version = [version for version in versions if version.name.split()[-1] == args.name][0]
```

Reply:
Good catch.

I think the nested `[]`s for indexing are a little disorienting for readability in a one-liner; Ruby has spoiled me, where is my good friend `Array#last`?

As an alternative, I can introduce a new method to check something like `if version.name in [args.name, args.project_id + ' ' + args.name]` or some `re` based check.